content/en/docs/guides/deploying-manifests-on-ibm-cloud/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Deploying Manifests on IBM Cloud"
```

Comment:
@yhwang I think you might have misunderstood what I am proposing.

We cannot continue to "endorse" specific installation methods on `kubeflow.org`, just as `kuberntes.io` does not say, "you should use GKE on Google Cloud" ,

To solve this, the community has discussed the following options in issues/community-calls:
1. Removing all distribution docs from `kubeflow.org` ([#2629](https://github.com/kubeflow/website/issues/2629))
2. Renaming the "distributions" which say "Kubeflow on XXXX" to not imply endorsement ([#2873](https://github.com/kubeflow/website/issues/2873), [#2967](https://github.com/kubeflow/website/issues/2967))

The community decided to initially do "option 2" by:
1. Distinguishing "Packaged" distributions from "Guides" to install `kubelfow/manifests`
2. Renaming "Kubeflow on XXXX" to "Deploying Manifests on XXXX" (removing the implied endorsement)

---

__NOTE 1:__ if you are saying that those IBM docs are actually a "packaged distribution" (and should live under `Distributions`), then it will need a name like the other "packaged" distributions (e.g. "Argoflow", "Arrikto MiniKF", "MicroK8S Kubeflow addon", etc")

__NOTE 2:__ none of this prevents IBM from using its other communication channels (not `kubeflow.org`) to endorse a method of installation

Reply:
I have same objections like @yhwang raised.  It is not correct to say "deploying kubeflow/manifests onto specific platforms" as platform installers are not using Kubeflow/manifests. It may or may not be same and can change in future. It should be left to organisations and individuals representing them are free to decide.  

1. " We cannot continue to "endorse" specific installation methods on kubeflow.org"  - What is being endorsed ? 
It is clearly stated in page - "Packaged distributions are developed and supported by their respective maintainers, the Kubeflow community does not currently endorse or certify any distribution."

Reply:
@johnugeorge 

Regarding your comment `"platform installers are not using Kubeflow/manifests"`, can you clarify which of the new "guides" created by this PR is not about deploying `kubeflow/manifests`:
- `Deploying Manifests on AWS`
- `Deploying Manifests on Azure`
- `Deploying Manifests on IBM Cloud`
- `Deploying Manifests on Nutanix Karbon`
- `Deploying Manifests on OpenShift`
- `Deploying Manifests with Kubeflow Operator`

When I look through the pages under each of those sections, all I see are guides on deploying/using the `kubeflow/manifests` on their respective platform, am I mistaken?

----

Regarding your comment `"It may or may not be same and can change in future. It should be left to organisations and individuals representing them are free to decide"`, obviously if a new distribution is created, it can be included under "Distributions", nothing about this PR prevents that.

This PR is about making the "current state" correct, there is no reason for the website to reflect what may-or-may-not happen in the future.

----

Regarding your question about `"What is being endorsed ?"`, I strongly believe any reasonable person browsing `kubeflow.org` who reads a page called `Kubeflow on AWS` will interpret that as the "community endorsed" method for installing Kubeflow on AWS. 

This is supported up by the large number of issues raised on `kubeflow/kubeflow` about what users think are "official" distributions for AWS/Azure/etc supported by the Kubeflow community.

Reply:
If IBM is forking at IBM/manifests, it seems like they should be a separate distribution instead.

Reply:
As discussed in https://github.com/kubeflow/website/pull/3004#issuecomment-971135029, I have updated the IBM page to:
- Deploying `ibm/manifests` on IBM Cloud